mne_connectivity/var.py

```diff
@@ -0,0 +1,406 @@
+from mne_connectivity.base import Connectivity, EpochConnectivity
+import os
+from pathlib import Path
+import shutil
+import numpy as np
+import scipy
+from scipy.linalg import sqrtm
+from tqdm import tqdm
+from sklearn.linear_model import Ridge
```

Comment:
can you nest the import? doing this you add a hard dependency. You deprecate code that did not require sklearn and now you require it. I doubt it's a big issue here but I wanted to comment to make you aware of such effects